private/extensions/busybox.bzl

```diff
@@ -14,12 +14,12 @@ def _busybox_impl(module_ctx):
     # To update amd64 busybox binary (#1014)
     # Get the latest commit hash from dist-amd64 branch of docker-library repo. You can also view it
     # at https://github.com/docker-library/official-images/blob/master/library/busybox
-    # Substitute it in the link: https://github.com/docker-library/busybox/raw/<latest-commit-hash>/latest/musl/busybox.tar.xz
+    # Substitute it in the link: https://github.com/docker-library/busybox/raw/<latest-commit-hash>/latest/musl/busybox.tar.gz
     # Update the sha256 value. Since github api doesn't give sha256 value, it can be obtained using sha256sum command.
     http_archive(
         name = "busybox_amd64",
-        sha256 = "77b216d55c6895ddb04a90f3025b5ce2480140da779fe3dca91303b135a1fefe",
-        urls = ["https://github.com/docker-library/busybox/raw/09ee80aedec1d8c604f104e8bec41ed19274620a/latest/musl/busybox.tar.xz"],
+        sha256 = "c69cfb3f9b07df5f5a73dafc28105e85211c399cfc336c59bfde5db1828c9f68",
+        urls = ["https://github.com/docker-library/busybox/raw/e5b5178110ca0332364a77d5eb4fff87b0e2ba3f/latest/musl/busybox.tar.gz"],
```

Comment:
it looks like this is now a symlink to `amd64/rootfs.tar.gz`, I took a look, and it's almost the same, there's a new utility in 1.37 called getfattr, but that's about it.

 but the behavior might be different when dealing with the different formats though. Anyway, these need to be updated to point to the right non symlinked file.